### PR TITLE
cast port and max_body_size_bytes to int, install watchdog

### PR DESCRIPTION
**Pull request by lukasvecerka23**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,7 +41,8 @@ dependencies = [
         "kazoo>=2.9.0,<3",
         "nest-asyncio==1.6.0",
         "python-dotenv==1.0.1",
-        "pyjwt==v2.10.0"
+        "pyjwt==v2.10.0",
+        "watchdog==6.0.0"
```
Comment (timthelion) [inline pyproject.toml:45]: watchdog should be in dev deps

Comment (lukasvecerka23) [reply pyproject.toml:45]: So should I install dev deps in bitswan editor? How should I do that when I'm installing it with `pip3 install git+<path to bitswan github commit>`?

Comment (timthelion) [reply pyproject.toml:45]: You install with `pip3 install ".[dev]"`

Comment (lukasvecerka23) [reply pyproject.toml]: Changed https://github.com/bitswan-space/BitSwan/pull/51/commits/94294b32c94e423ee43c0e4a2774888d2e31785a

**Pull request merged**